Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Feedback.hs

```diff
@@ -171,6 +181,23 @@ feedback request personId = do
               ]
        in TL.toStrict $ TLE.decodeUtf8 $ A.encode jsonValue
 
+    mkTicket :: Person.Person -> Maybe Text -> Ticket.RideInfo -> Maybe Text -> Text -> Text -> Ticket.CreateTicketReq
+    mkTicket person phoneNumber info mediaFileUrl disposition queue = do
+      Ticket.CreateTicketReq
+        { category = "Customer Feedback",
+          subCategory = Just "L0 Feedback",
+          issueId = Nothing,
+          issueDescription = "THIS IS A TEST TICKET PLEASE IGNORE IT - " <> fromMaybe "" request.feedbackDetails,
```

Comment:
What why test?

Reply:
pushed it accidentally after testing :{
Removing